### DISCUSSION TITLE
keyboard shortcut for logout

### DISCUSSION THREAD

**Discussion by niels-borne** [FreshRSS/FreshRSS#3826] [Ideas]
Hi,

a keyboard shortcut for logging out would be useful to me : to my taste the current way is a bit slow and not very convenient.

Again, thank you for your wonderful work !

**Comment by Alkarex** [comment]
I am imagining that the most typical case is to leave the session open, but patch welcome

**Reply by niels-borne** [reply]
Thank you, but in fact you overestimate me, I'm really a beginner when it comes to code.
